bash/sudo_bang.md

```diff
@@ -0,0 +1,35 @@
+# Useful *Bang*
+
+I found another great thing bash console provides for the lazy programmers ;) Most of the time we need to repeat the commands we entered in linux terminal and many hardworking engineers typing it again and again ;)
+
+Linux provides many tools which remembers history and on top of that we can make our life easy.
+
+Usually i work on servers and whenever try to fire few root access commands and it dont allow me!! -_-,feels insulting. Anyway sudo !! is here to rescue you :)
```

Comment:
Can we rephrase this paragraph? Maybe: "Bang commands make it easy to re-run commands as root if you forget to use `sudo`."